starter/multiwoz/hotel/schema.tt

```diff
@@ -24,7 +24,10 @@ class @uk.ac.cam.multiwoz.Hotel
                      preposition=["in the #", "in the # area", "in the # part of town",
                                   "near the #", "near the # area", "near the # part of town",
                                   "around the #", "around the # area", "around the # part of town",
-                                  "in the # of cambridge", "in # cambridge"]
+                                  "in the # of cambridge", "in # cambridge"],
+                     projection_pronoun=["where", "which part of town", "which area", "which part of cambridge"],
+                     verb_projection=["locate"],
```

Comment:
wait isn't this passive_verb_projection instead?

Reply:
oops, mistake